.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -10,5 +10,5 @@ labels: 'questions'
 
 Please contact us instead. We are active in gitter and twitter:
 
-- [Gitter](https://gitter.im/reactioncommerce/reaction)
+- [Gitter](https://gitter.im/mailchimp-open-commerce/api-server)
```

Comment:
Are we changing our Gitter url?

Reply:
I was hoping that they would update once we rename the org/repo. I can try to get a confirmation on this one.